[ASI] Image details, editing, search and metadata (#649)
magento/merchdocs: src/cms/media-gallery-asset-management.md

    ![]({% link images/images/enhanced-media-gallery/asset-actions.png %}){: .zoom}
    _Asset Actions_

    The asset details will be displayed on a slide panel. They include the information where the asset is being used:

   - **Categories**
   - **Products**
   - **Pages**
   - **Blocks**

    ![]({% link images/images/enhanced-media-gallery/asset-details.png %}){: .zoom}
    _Asset Details_

>     Click **Used In** links to see the details. The grid in the following example shows all categories where specific asset is used.

Reviewer: ```suggestion
    Click the **Used In** links to see the details. The grid in the following example shows all categories where a specific asset is used.
```

    ![]({% link images/images/enhanced-media-gallery/asset-category-grid.png %}){: .zoom}
    _Category Grid_

>     It is also possible to delete the asset from the **View Details** section.

Reviewer: ```suggestion
    It is also possible to delete the asset from the _View Details_ section.
```

## Edit an asset

1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.

1. Click the three dots below the asset (![]({% link images/images/enhanced-media-gallery/three-dots-btn.png %})), then click <span class="btn">Edit</span>.

    ![]({% link images/images/enhanced-media-gallery/edit-asset.png %}){: .zoom}
    _Edit Asset_

> 1. You can change following fields though **Edit Image** form.

Reviewer: ```suggestion
1. If needed, change and of the following metadata values:
```

   - **Title**
   - **Description**
   - **Tags/Keywords**

>    Please note, that this data will be saved to the data base as well as to the file metadata itself. Currently, **XMP** and **IPTC* formats are supported.

Reviewer: ```suggestion
   This data will be saved to the data base as well as to the file metadata itself. Currently, XMP and IPTC formats are supported.
```

    ![]({% link images/images/enhanced-media-gallery/bulk-delete-asset-confirmation.png %}){: .zoom}
    _Delete Confirmation_

## Search for assets

1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.

> 1. Use **Search by keywords** input to perform image search by keywords/tags.

Reviewer: ```suggestion
1. Use the **Search by keywords** input to perform image search by keywords/tags.
```

    The search in the following example finds assets that contain a specific tag (`mountain`).

    ![]({% link images/images/enhanced-media-gallery/asset-search.png %}){: .zoom}
    _Asset Search_

> 1. See [Edit an asset][edit-asset] section to learn how you can update image tags.

Reviewer: This really isn't a step

```suggestion
{:.bs-callout-info}
See the [Edit an asset][edit-asset] section to learn how you can update image tags.
```
